tests/end_to_end/test_fim/test_fim_linux/test_fim_linux.py

```diff
@@ -22,7 +22,7 @@
 @pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
 def test_fim(configure_environment, metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Comment:
```suggestion
def test_fim_linux(configure_environment, metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Reply:
Done in 32249dfa1d732e79983d294b7848c8f12e53a5cf